server.js

```diff
@@ -20,6 +22,8 @@ app.post("/login", validateLogin(), user.loginUser);
 
 app.post("/users/:id/responders", middleware.checkToken, user.addResponders);
 
+app.post("/users/:id/notificationToken", user.addPushToken);
```

Comment:
this is picky but can we do
```suggestion
app.post("/users/:id/notification-token", user.addPushToken);
```
to be consistent